reference/functions/classfiltercsv.markdown

```diff
@@ -8,7 +8,7 @@ tags: [reference, csv, data functions, functions, classfiltercsv]
 [%CFEngine_function_prototype(filename, has_header, class_column, optional_sort_column)%]
 
 **Description:**
-Parses CSV data from the file `filename`, and returns a data variable that is
+Parses CSV data from an [RFC 4180](http://www.ietf.org/rfc/rfc4180.txt) compliant file `filename`, and returns a data variable that is
```

Comment:
I would almost just remove the Note and put `(CRLF line endings required)` after `compliant`.